alpha/lib/enums/PermissionName.php

```diff
@@ -236,6 +236,7 @@
 	
 	const FEATURE_SHOW_FLASH_STUDIO = 'FEATURE_SHOW_FLASH_STUDIO';
 	const FEATURE_SHOW_HTML_STUDIO = 'FEATURE_SHOW_HTML_STUDIO';
+	const FEATURE_ENABLE_USAGE_DASHBOARD = 'FEATURE_ENABLE_USAGE_DASHBOARD';
```

Comment:
@tan-tan-kanarek, we are not trying to keep all available permissions organized in one place?
cc @erankor